packages/components/src/icon-library.tsx

```diff
@@ -95,12 +94,12 @@ import { ReactComponent as trashCircleFilled } from './icons/trash-circle-filled
 import { ReactComponent as warningInfo } from './icons/warning-info.svg';
 import { ReactComponent as wdl } from './icons/wdl.svg';
 
-const fa = _.curry((shape, { size, ...props }) =>
-  h(FontAwesomeIcon, _.merge({ icon: shape, style: { height: size, width: size } }, props))
-);
-const custom = _.curry((shape, { size, ...props }) =>
-  h(shape, _.merge({ 'aria-hidden': true, focusable: false, style: { height: size, width: size } }, props))
-);
+const fa = _.curry((shape, { size, ...props }) => (
+  <FontAwesomeIcon {..._.merge({ icon: shape, style: { height: size, width: size } }, props)} />
+));
+const custom = _.curry((Shape, { size, ...props }) => (
```

Comment:
Where is capital S Shape coming from? I don't see it imported or defined anywhere.

Reply:
It's the first argument to the function.

Reply:
My bad. Thank you.